panel/applets/places-indicator/Translatables.vala

```diff
@@ -0,0 +1,21 @@
+/*
+ * This file is part of budgie-desktop
+ *
+ * Copyright (C) 2015-2016 Solus Project
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+
+const string PLACES_STRING = _("Places");
+const string OPEN_LOCATION_STRING = _("Open location");
+const string REMOVABLE_DRIVES_STRING = _("Removable drives");
+const string NETWORK_FOLDERS_STRING = _("Network folders");
+const string EJECT_STRING = _("Eject");
+const string UNMOUNT_STRING = _("Unmount");
+const string SAFE_REMOVE_STRING = _("You can now safely remove");
```

Comment:
Sorry, first time working with translatables. You are right about them being out of context. I will gladly change this.